Update postgres-custom-metric-collection-explained.md (#26545)
DataDog/documentation: content/en/integrations/faq/postgres-custom-metric-collection-explained.md

`custom_queries` has the following options:

`metric_prefix`

> : **Required**: Yes </br/> 

Reviewer: ```suggestion
: **Required**: Yes <br/> 
```

Each metric starts with the chosen prefix.

`query`
: **Required**: Yes <br/> 
The SQL to execute. It can be a simple statement or a multi-line script. All of the rows of the results are evaluated. Use the pipe if you require a multi-line script.     

`columns`
: **Required**: Yes <br/> 

> A list representing each column ordered sequentially from left to right. There are 2 required pieces of data:<br>  - **`name`**: The suffix to append to the metric_prefix to form the full metric name. If the `type` is specified as `tag`, the column is instead applied as a tag to every metric collected by this query.<br>  - **`type`**: The submission method (for example, `gauge`, `count`, `rate`, etc.). This can also be set to `tag` to tag each metric in the row with the name and value (`<name>:<row_value>`) of the item in this column. 

Reviewer: ```suggestion
A list representing each column ordered sequentially from left to right. There are 2 required pieces of data:<br>  - **`name`**: The suffix to append to the `metric_prefix` to form the full metric name. If the `type` is specified as `tag`, the column is instead applied as a tag to every metric collected by this query.<br>  - **`type`**: The submission method (for example, `gauge`, `count`, `rate`, etc.). This can also be set to `tag` to tag each metric in the row with the name and value (`<name>:<row_value>`) of the item in this column. 
```

| query         | Yes      | This is the SQL to execute. It can be a simple statement or a multi-line script. All of the rows of the results are evaluated. Use the pipe if you require a multi-line script.                                                                                                                                                                                                                                                                                                                                                                                                                              |

> | columns       | Yes      | This is a list representing each column ordered sequentially from left to right.<br><br>There are 2 required pieces of data:<br>  - **`name`**: This is the suffix to append to the metric_prefix to form the full metric name. If the `type` is specified as `tag`, the column is instead applied as a tag to every metric collected by this query.<br>  - **`type`**: This is the submission method (ex: `gauge`, `count`, `rate`, etc.). This can also be set to `tag` to tag each metric in the row with the name and value (`<name>:<row_value>`) of the item in this column.                           |

Reviewer: ```suggestion
| columns       | Yes      | This is a list representing each column ordered sequentially from left to right.<br><br>There are 2 required pieces of data:<br>  - **`name`**: This is the suffix to append to the metric_prefix to form the full metric name. If the `type` is specified as `tag`, the column is instead applied as a tag to every metric collected by this query.<br>  - **`type`**: This is the submission method (for example, `gauge`, `count`, `rate`, etc.). This can also be set to `tag` to tag each metric in the row with the name and value (`<name>:<row_value>`) of the item in this column.                           |
```

| tags          | No       | A list of static tags to apply to each metric.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |

### Notes

> - At least one of the items in defined `columns` should be a metric type (`gauge`, `count`, `rate`, etc.). To find more information about metric submission from an Agent Check, see the [Metrics Type][7] doc.

Reviewer: ```suggestion
- At least one of the items in defined `columns` should be a metric type (`gauge`, `count`, `rate`, etc.). For more information about metrics submission from an Agent Check, see [Metrics Types][7].
```
